### ISSUE TITLE
CMake fails looking for GLEW::glew

### ISSUE THREAD

**Issue by thewierdnut** [endless-sky/endless-sky#10519]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Trying to run cmake on debian bookworm, it tells me it can't find `GLEW::glew` when using VC_PKG

### Steps to Reproduce

Run `cmake .. -DCMAKE_BUILD_TYPE=Debug -DES_USE_VCPKG=1

### Expected Behavior

cmake finishes.

### Screenshots

```
CMake Error at CMakeLists.txt:173 (target_link_libraries):
  The link interface of target "ExternalLibraries" contains:

    GLEW::glew

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.
```

### Operating System

debian bullseye

### Game Source

Built from source

### Game Version

v0.10.9

### Additional Information

Running with `-DES_USE_VCPKG=0` works fine. Its only when using VCPKG that it breaks.

**Comment by warp-core** [comment]
Does it work if you undo https://github.com/endless-sky/endless-sky/pull/9978?

**Comment by thewierdnut** [comment]
Yes. We had this discussion on discord, I just figured I would post an issue so it isn't forgotten.
